tests/functional/Item_Rack.php

```diff
@@ -490,4 +491,295 @@ public function testAdd()
             ])
         )->isGreaterThan(0);
     }
+
+    /**
+     * Test for update items models into rack
+     */
+    public function testAlterModels()
+    {
+        // Create a ComputerModel
+        $model1 = new \ComputerModel();
+        $this->integer(
+            (int)$model1->add([
+                'required_units'  => 1,
+                'depth'           => 1,
+                'is_half_rack'    => 0,
+                'entities_id'     => 0,
+            ])
+        )->isGreaterThan(0);
```

Comment:
Its not mandatory to use it but in case you don't know we have a `createItem` method that does the `isGreaterThan(0)` + some others checks for you.

We also have `createItems` and `updateItem`.